doc/source/notebooks/understanding/models.pct.py

```diff
@@ -199,14 +195,29 @@ def closure():
 #
 # In this very simple demo, we'll implement linear multiclass classification.
 #
-# There are two parameters: a weight matrix and a bias (offset). The key thing to implement the `log_likelihood` method, which returns a TensorFlow scalar that represents the (log) likelihood. You can use parameter objects inside `log_likelihood`.
+# There are two parameters: a weight matrix and a bias (offset). You can use
+# Parameter objects directly, like any TensorFlow tensor.
+#
+# The training objective depends on the type of model; it may be possible to
```

Comment:
Is this still up-to-date with the new Mixin classes?

Reply:
If so, wouldn't it be nice to mention them? Also, maybe say something about the `training_loss_closure`?

Reply:
done - added a paragraph and a link to the intro_to_gpflow2 notebook which contains more details.